libminifi/include/utils/StringViewUtils.h

```diff
@@ -0,0 +1,68 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <cctype>
+#include <algorithm>
+#include "StringView.h"
+#include "utils/OptionalUtils.h"
+
+namespace org {
+namespace apache {
+namespace nifi {
+namespace minifi {
+namespace utils {
+
+struct StringViewUtils {
+  static inline bool equalsIgnoreCase(StringView left, StringView right) {
```

Comment:
String views are essentially strings so I would rather move these utils to StringUtils.h.

Reply:
I wouldn't call string views, strings, maybe readonly-strings, but even that is a stretch, rather strings are special string views that own their content, moreover I think StringUtils.h is already pretty long so if anything I would chop off the "encoding/decoding" part of the StringUtils.h into a separate file